tracer/src/Datadog.Trace/Agent/AgentTransportStrategy.cs

```diff
@@ -47,7 +47,9 @@ internal static class AgentTransportStrategy
         {
             case TracesTransportType.WindowsNamedPipe:
                 // intentionally using string interpolation, as this is only called once, and avoids array allocation
+#pragma warning disable DDLOG004 // Message templates should be constant
```

Comment:
How come you disabled the warning here? It seems like we could rewrite this to use a Log.Information<string, string, int> overload

Reply:
Good point 😄 Yeah, shouldn't be needed here, despite what the line above says. cheers 👍

Reply:
Fixed in https://github.com/DataDog/dd-trace-dotnet/pull/3785/commits/aae37a38d7293c694d024e0d6fc6b114832a6476, thanks!